app/services/projects/copy/work_packages_dependent_service.rb

```diff
@@ -137,7 +137,6 @@ def copy_work_package_attribute_overrides(source_work_package, parent_id, user_c
         project: target,
         parent_id: parent_id,
         version_id: work_package_version_id(source_work_package),
-        assigned_to_id: work_package_assigned_to_id(source_work_package),
```

Comment:
Why was this removed? Is that wrong? Either way it has nothing to do with the autologin cookie. If it is a bug that you fixed while you were at it it would make sense to at least put it into a separate commit with a short explanation.

Reply:
Ah thanks, that was by mistake